tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,570 @@
+        assert (
+            new_node == secondary_node
+        ), f"Floating monitor did not migrate: expected {secondary_node}, got {new_node}"
+
+        logger.test_step("Verify quorum maintained")
+        ceph_cluster = CephCluster()
+        quorum_status = ceph_cluster.toolbox.exec_cmd_on_pod(
+            "ceph quorum_status --format json"
+        )
+        quorum_data = json.loads(quorum_status)
+        quorum_names = quorum_data.get("quorum_names", [])
+        logger.info(f"Quorum members during failover: {quorum_names}")
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)